The next release of cryptography won't support Python 2.  This fact is echoed
to the user each time this library is imported, which is generally not desired.

Not intended for upstream.

--- cryptography-3.3.1/src/cryptography/__init__.py
+++ cryptography-3.3.1/src/cryptography/__init__.py
@@ -4,9 +4,6 @@
 
 from __future__ import absolute_import, division, print_function
 
-import sys
-import warnings
-
 from cryptography.__about__ import (
     __author__,
     __copyright__,
@@ -17,7 +14,6 @@ from cryptography.__about__ import (
     __uri__,
     __version__,
 )
-from cryptography.utils import CryptographyDeprecationWarning
 
 
 __all__ = [
@@ -30,12 +26,3 @@ __all__ = [
     "__license__",
     "__copyright__",
 ]
-
-if sys.version_info[0] == 2:
-    warnings.warn(
-        "Python 2 is no longer supported by the Python core team. Support for "
-        "it is now deprecated in cryptography, and will be removed in the "
-        "next release.",
-        CryptographyDeprecationWarning,
-        stacklevel=2,
-    )
